### PR TITLE
Add link to GitHub repo on about flyout

### LINKED ISSUE #582: Add link to GitHub repo on about flyout
**Problem Statement**
There is no way to navigate to the project from within the Calculator app.

**Evidence or User Insights**

**Proposal**
By adding a link to the GitHub repo on the about flyout, people will know that it's open source and that they can create issues, PRs and more.

**Goals**
Improve discoverability of the GitHub project from within Calculator.

**Non-Goals**

**Low-Fidelity Concept**
Add this string to the about dialog after the "Send feedback" button:
> To learn how you can contribute to Windows Calculator, check out the project on [GitHub](https://go.microsoft.com/fwlink/?linkid=2099939).

image

**Requested Assignment**
I'm just suggesting this idea.  I don't want to implement it.

> Note: Copied/split from #543

### PR DESCRIPTION
**Pull request by chaitanyamehta**
## Fixes #582
Display link to the GitHub repo on the about flyout. Implementation is similar to link shown in currency converter mode when application is offline. (Offline. Please check your%HL%Network Settings%HL%)

### Description of the changes:
- Add  learn more and contribute string in en-us resource.
- Display link below feedback button in About Flyout.
- Disable ScrollViewer.HorizontalScrollBarVisibility so that text wraps properly.

About Page

### How changes were validated:
- Manually tested.

### CODE REVIEW

File: src/Calculator/AboutFlyout.xaml [hunk 1]
```diff
@@ -72,5 +73,21 @@
                     Click="FeedbackButton_Click"/>
 
         </StackPanel>
+        <RichTextBlock x:Name="AboutFlyoutContribute"
+                       Grid.Row="2"
+                       Grid.ColumnSpan="2"
+                       Margin="12,10,12,6"
```
Comment (grochocki) [inline src/Calculator/AboutFlyout.xaml:79]: Nit: I think if we bump this up to 18, it will be vertically offset to match space between the hyperlinks and the feedback button above.

Comment (chaitanyamehta) [reply src/Calculator/AboutFlyout.xaml]: Have updated GitHub link margin. Below are two screenshots of about page.

About page in full screen mode.
About Page

About page with calculator restored down to minimum size.
About Page

File: src/Calculator/Views/MainPage.xaml [hunk 2]
```diff
@@ -28,6 +28,7 @@
         <Style x:Key="AboutFlyoutPresenterStyle" TargetType="FlyoutPresenter">
             <Setter Property="IsTabStop" Value="False"/>
             <Setter Property="AutomationProperties.AccessibilityView" Value="Raw"/>
+            <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Disabled"/>
```
Comment (greedyAI) [inline src/Calculator/Views/MainPage.xaml:31]: Why is this necessary? There is no ScrollViewer in the AboutFlyout.

Comment (chaitanyamehta) [reply src/Calculator/Views/MainPage.xaml:31]: The default control template of FlyoutPresenter contains a ScrollViewer.

Control Template

This is what it looks like if the ScrollViewer property setter is removed:
About Page with Scroll

Comment (greedyAI) [reply src/Calculator/Views/MainPage.xaml:31]: Ah, ok. That makes sense. Good work!

**Review by greedyAI — APPROVED** [review]

**Review by joseartrivera — APPROVED** [review]

File: src/Calculator/Resources/en-US/Resources.resw [hunk 3]
```diff
@@ -2717,6 +2717,10 @@
     <value>© %1 Microsoft. All rights reserved.</value>
     <comment>{Locked="%1"}. Copyright statement, displayed on the About panel. %1 = the current year (4 digits)</comment>
   </data>
+  <data name="AboutFlyoutContribute" xml:space="preserve">
+    <value>Windows Calculator is made available under the MIT license. To learn more and contribute, check out the project on %HL%GitHub%HL%.</value>
```
Comment (grochocki) [inline src/Calculator/Resources/en-US/Resources.resw:2721]: Changes look great! One last tweak -- can we update this string to "To learn how you can contribute to Windows Calculator, check out the project on GitHub"

Comment (chaitanyamehta) [reply src/Calculator/Resources/en-US/Resources.resw]: Updated

About Page

Comment (grochocki) [reply src/Calculator/Resources/en-US/Resources.resw]: Definitely my typo, but can we add back the period?

Comment (chaitanyamehta) [reply src/Calculator/Resources/en-US/Resources.resw]: Sure

**Review by grochocki — APPROVED** [review]

**Comment by grochocki** [comment]
Perfect! Thanks again, @chaitanyamehta.
